import org.omg.PortableInterceptor.SYSTEM_EXCEPTION;

import java.util.*;
import java.io.*;

public class Main {
    public static void main(String[] args) {

//        Four_Sum_II_454 f = new Four_Sum_II_454();
//
//        int[] nums1 = {100,300};
//        int[] nums2 = {-100,400};
//        int[] nums3 = {-1,1};
//        int[] nums4 = {1,1};
//
//        int a = f.fourSumCount(nums1,nums2,nums3,nums4);
//        String arr = "zhang";
//        char[] s = arr.toCharArray();
//        Arrays.sort(s);
//        String x = String.valueOf(s);



//        HashMap<Integer,Integer> map = new HashMap<Integer, Integer>() ;
//        map.put(1,1);
//       System.out.print(map.get(1) ==1);


    }
}


